.vale/fixtures/OpenShiftAsciiDoc/MergeConflictMarkers/testvalid.adoc

```diff
@@ -0,0 +1,4 @@
+//Module is included in...
+
```

Comment:
You could add something that might be close to causing your rule to flag, but doesn't. For example add 8 > symbols. Or `|=======`

Reply:
Thanks. I went with:

```suggestion
//testvalid
|=======

//testvalid
======

//testvalid
========
```